Apps/WebClient/src/ClientApp/src/models/dateWrapper.ts

```diff
@@ -188,10 +188,10 @@ export class DateWrapper {
      * Difference between two Dates in milliseconds
      * @param other Other date to compare to
      * @param unit (optional) unit of comparision (year, hour, minute, second...)
-     * @returns the difference in milliseconds
+     * @returns the difference with the Duration object
      */
-    public diff(other: DateWrapper, unit?: DurationUnit): number {
-        return this.internalDate.diff(other.internalDate, unit).milliseconds;
+    public diff(other: DateWrapper, unit?: DurationUnit): Duration {
```

Comment:
This fix seems not to be a part of the scope of this PR.

Reply:
not a bug just made the method more useful.